pkg/windows/windows.go

```diff
@@ -591,6 +597,21 @@ func (vm *windows) createDirectories() error {
 // removeDirectories removes all directories created as part of the configuration process
 func (vm *windows) removeDirectories() error {
 	vm.log.Info("removing directories")
+	existingFiles, err := getFilesToTransfer()
+	if err != nil {
+		return errors.Wrapf(err, "error getting list of files transferred to the instance")
+	}
+	for src, dest := range existingFiles {
+		filePath := dest + "\\" + filepath.Base(src.Path)
+		if out, err := vm.Run(rmFileCmd(filePath), true); err != nil {
```

Comment:
Should we consider retrying a few times? I am worried that given the amount network issues we see executing commands over SSH we are going to have to reconcile a few times before all files are deleted. Or should we introduce a script that does the file deletion and HSN network removal?